Cloud_Region_Metadata_specification.md

```diff
@@ -1,119 +1,421 @@
+
+### Infrastructure metrics
+
+#### Geographic coordinates
+
+Cloud service providers shall provide approximate geographic coordinates (latitude and longitude) for each cloud region to enable grid region mapping.
+
+#### Grid region identification
+
+Cloud service providers shall identify the electrical grid region(s) serving each cloud region using standard grid operator identifiers, where available.
+
+#### Climate zone
+
+Cloud service providers should provide climate zone classification according to international standards to enable cooling efficiency assessment.
+
+### Operational metrics
+
+#### Service availability
+
+Cloud service providers should report annual average service availability percentages for core services in each region.
+
+#### Data residency
+
+Cloud service providers shall specify data residency requirements and restrictions for each region.
+
+---
+
+## Data access requirements
+
+Cloud providers shall:
+
+a) make the cloud region metadata accessible to customers in a machine-readable format;
+
+b) provide appropriate identifiers (em-zone-id and wt-region-id) to enable customers to access real-time data from third-party providers;
+
+c) document any changes to the metadata format or calculation methodologies in subsequent releases.
+
+---
+
+## Conformance
+
+### General
+
+Conformance to this document requires implementation of all normative requirements specified in Clauses 4 through 7 and Annex A.
+
+### Conformance levels
+
+This document defines two levels of conformance:
+
+a) **Level 1 (Basic conformance)**: Implementation of mandatory metadata parameters as specified in Annex A and basic reporting capabilities.
+
+b) **Level 2 (Advanced conformance)**: Implementation of all Level 1 requirements plus real-time API access and advanced data quality validation.
+
+### Conformance declaration
+
+Cloud service providers claiming conformance to this document shall provide a conformance declaration that specifies:
+
+a) the conformance level achieved (Level 1 or Level 2);
+
+b) the scope of implementation (regions covered, services included);
+
+c) the date of conformance assessment;
+
+d) identification of any limitations or exceptions.
+
+---
+
+## Annex A (normative) Mandatory metadata parameters
+
+### A.1 Level 1 conformance requirements
+
+Cloud service providers claiming Level 1 conformance shall provide the following mandatory parameters for each cloud region:
+
+**Table A.1 — Mandatory metadata parameters for Level 1 conformance**
+
+| Parameter | Unit | Update Frequency | Required |
+|-----------|------|------------------|----------|
+| year | numeric | Annual | Yes |
+| cloud-provider | string | Static | Yes |
+| cloud-region | string | Static | Yes |
+| location | string | Static | Yes |
+| geolocation | decimal degrees | Static | Yes |
+| power-usage-effectiveness | ratio | Annual | Yes |
+| grid-carbon-intensity | gCO₂eq/kWh | Annual | Yes |
+| provider-cfe-annual | proportion | Annual | Yes |
```

Comment:
hi @seanmcilroy29  - I appreciate I missed this on the last call, that new levels of conformance had been introduced to this - both A1. (mandatory) and A2. (optional).

Was this what came out of the conversation in #109  when speaking to folks at the ISO? I'm asking in the context of people who might build tooling for future validation against different levels of conformancy.